packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -274,6 +274,49 @@ export interface RaiseHandCallFeatureState {
   localParticipantRaisedHand?: RaisedHandState;
 }
 
+/* @conditional-compile-remove(together-mode) */
+/**
+ * State only version of {@link @azure/communication-calling#TogetherModeCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for raised hands on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ * @alpha
+ */
+export interface TogetherModeCallFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeCallFeature.togetherModeStream}.
+   */
+  stream?: TogetherModeStreamState;
```

Comment:
Is there ever a chance there would be more than one together mode stream per call? (Feels unlikely but hard to adapt this API in future if there ever was)

Reply:
Yes. I am working to update this and would include the change in my next commit